test/gtest/conv_api_strided_tensors.cpp

```diff
@@ -139,7 +139,6 @@ class ConvStridedTensors : public ::testing::Test
     std::vector<float> h_output;
 };
 
-// This test should be replaced when strided tensors are fully implemented
```

Comment:
FYI This test was needed to verify that the library returns an error if unpacked tensors are used for convolutions.

Reply:
Are you OK with this change or should we remove the test altogether?

Reply:
This test is no longer needed and can be removed. We need to make a test for unpacked tensors instead.

Reply:
True. We don't have a public API test for non-packed tensors. I have modified reference convolution tests to generate non-packed strides in this PR and #2418 will add tests for 3D convolution, but those target specific solvers. In general, it seems we lack public API tests (or is it my misconception?). A lot of tests seem to use internal APIs.